### PR TITLE
Added a new config option to set a custom notification interval

### LINKED ISSUE #35: make notification interval configurable

### PR DESCRIPTION
**Pull request by Sempiternity87**
Added a configuration option for the notification interval. Based on the original code, I've set the default value to 5 minutes, but this should allow someone to only get notifications every hour (60 minutes), or day (1440), or any other feasible interval.

It's possible that this may spawn another issue to allow setting the TimeUnit as well.

I don't know how to build it to test myself, so I totally understand if this should be put in a different branch. I'll leave that to your discretion.

### CODE REVIEW

**Review by shayaantx — CHANGES_REQUESTED** [review]
can you rebase your commits into one and bump the version.txt file

**Comment by Sempiternity87** [comment]
Do you want me to bump the version to 5.6.5, or keep it at 5.6.4 after syncing?

**Comment by shayaantx** [comment]
> Do you want me to bump the version to 5.6.5, or keep it at 5.6.4 after syncing?

5.6.5

File: README.md [hunk 1]
```diff
@@ -194,6 +194,7 @@ botdarr:
 | EXISTING_ITEM_PATHS_BLACKLIST | If you want content to NOT appear in searches against your library, you can list blacklisted paths here in comma delimited form, and they will be ignored when building up responses | no |
 | MAX_DOWNLOADS_TO_SHOW | The max number of downloads to show. If you set this to any value less than or equal to 0, no downloads will show | yes | 20 |
 | MAX_RESULTS_TO_SHOW | The max number of results to show per search command. If you set this to any value less than 0, the bot won't startup | yes | 20 | 
+| NOTIFICATION_INTERVAL | The number of minutes between each notification. If you set this to any value less than 0, the bot won't startup | yes | 20 |
```
Comment (shayaantx) [inline README.md:197]: technically you are just logging the exception, so the bot will startup, but I don't really care :)

**Review by shayaantx — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (Sempiternity87) [reply README.md:197]: Yeah, I wasn't sure about this. I tried to see where the MAX_DOWNLOADS_TO_SHOW and MAX_RESULTS_TO_SHOW would have thrown an error as well, but I didn't see anything specific. I didn't know if the delay function would have a problem with a negative, but I figured it was at least safer to say it wouldn't work, than to have it not work with no explanation.

Comment (Sempiternity87) [reply README.md:197]: Oh, I don't know how we both missed this: the default value specified here says 20, but it should be 5, as that's what it is in the ApiRequests.java.

**Pull request merged**